tensorflow/lite/kernels/internal/reference/log_softmax.h

```diff
@@ -230,7 +228,7 @@ inline void LogSoftmaxQuantized(const SoftmaxParams& params,
             gemmlowp::RoundingDivideByPOT(
                 (input_diff_in_q5 - log_sum_of_exps_in_q5),
                 31 - kInputIntegerBits - kOutputIntegerBits) +
-            params.zero_point;
+            kMaxT8;
```

Comment:
This is a slightly surprising change - I just wanted to check that it was deliberate?

Reply:
Yes that is intentional.  I saved 4 bytes during Init phase by not storing the zero point.  The correct zero point for output is checked during the Prepare phase.  So the zero point will always match kMaxT8 when calling this template.